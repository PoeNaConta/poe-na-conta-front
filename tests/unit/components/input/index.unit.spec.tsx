import React from 'react';
import Input from '@components/input';
import { render } from '@testing-library/react';

describe('Input', () => {
  it('should render basic input', () => {
    const { container } = render(<Input />);

    expect(container).toMatchSnapshot();
  });

  it('should render input with label', () => {
    const { container } = render(<Input />);

    expect(container).toMatchSnapshot();
  });
});
