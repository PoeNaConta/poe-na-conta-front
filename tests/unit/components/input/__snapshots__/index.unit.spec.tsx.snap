// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Input > should render basic input 1`] = `
<div>
  <div
    class="lds--input__wrapper"
  >
    <label
      class="lds--text lds--text--regular lds--text--md lds--text--default"
    />
    <input
      class="lds--input__input"
    />
  </div>
</div>
`;

exports[`Input > should render input with label 1`] = `
<div>
  <div
    class="lds--input__wrapper"
  >
    <label
      class="lds--text lds--text--regular lds--text--md lds--text--default"
    />
    <input
      class="lds--input__input"
    />
  </div>
</div>
`;
